tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -214,6 +214,16 @@ public void syncOverrideLoadMetadata() throws Exception {
     checkGetStatus("/mnt/dir1/dirA/file", options, true, true, 1);
   }
 
+  @Test
+  public void loadMetadataListDir() throws Exception {
```

Comment:
What is this behavior before the change?

Reply:
when doing a ls dir triggering loadmetadata , it does ufs `ls dir` and ufs `ls dir/dira`, `dir/dirb`, which should not be necessary.